pandas/tests/frame/test_repr.py

```diff
@@ -71,9 +71,12 @@ def test_assign_index_sequences(self):
         repr(df)
 
         # this travels an improper code path
+        # 60925 should raise when one of index's items is a list and others are not
         index[0] = ["faz", "boo"]
-        df.index = index
-        repr(df)
+        msg = "Index names must all be hashable, or all lists to make MultiIndex"
+        with pytest.raises(ValueError, match=msg):
+            df.index = index
+            repr(df)
```

Comment:
This looks like it should raise on the first line, is that right? If so, can you remove `repr(df)`.